manifests/kustomize/components/postgresql/helm-chart-generator.yaml

```diff
@@ -13,16 +13,42 @@ skipTests: true
 valuesInline:
   nameOverride: *name
   fullnameOverride: *name
+  postgresqlSharedPreloadLibraries: pg_cron
   global:
     postgresql:
       auth:
+        postgresPassword: password
         username: root
         password: password
         database: main
+  image:
+    registry: public.ecr.aws
+    repository: n4e0e1y0/postgres
+    tag: 16.1
+    pullPolicy: Always
```

Comment:
I would probably change this to `IfNotPresent` to avoid using bandwidth or slowing the startup process if its already cached.

Reply:
done